infra/gcp/bash/lib_gcr.sh

```diff
@@ -73,14 +77,19 @@ function ensure_gcr_repo() {
         host=$(gcr_host_for_region "${region}")
         local image="ceci-nest-pas-une-image"
         local dest="${host}/${project}/${image}"
-        docker pull k8s.gcr.io/pause
-        docker tag k8s.gcr.io/pause "${dest}"
+        docker pull us-central1-docker.pkg.dev/k8s-artifacts-prod/images/pause
+        docker tag us-central1-docker.pkg.dev/k8s-artifacts-prod/images/pause "${dest}"
```

Comment:
this is a good choice because our CI generally runs here*, so we avoid egress.

if we run this in say, cloud build, then we should probably try to use $current region, but that's tricky because we're not currently in _all_ regions. just using a known region is probably easiest for now.

* sometimes if we spin up an e2e cluster it may run in AWS even, but generally speaking kubernetes infra is in central-1 if not multi-region, the CI environment itself for example is in central-1.